app/forms/provider_interface/provider_relationship_permissions_setup_wizard.rb

```diff
@@ -69,15 +69,12 @@ def permissions_for_persistence
 
     # Steps are
     # 1. provider_relationships
```

Comment:
Does this comment need updating (e.g. `:start`, `:organisations`)?

Reply:
I'll remove `:start` as it's redundant

Reply:
@mnacos housekeeping commit to remove redundant steps https://github.com/DFE-Digital/apply-for-teacher-training/pull/3222/commits/0c20cdc683867ada716f41c22bd050132202834c